content/en/docs/reference/config/analysis/ist0105/index.md

```diff
@@ -3,7 +3,7 @@ title: IstioProxyVersionMismatch
 layout: analysis-message
 ---
 
-This message is emitted regarding a pod when:
+This message occurs regarding a pod when:
 
 * Automatic sidecar injection is enabled (default enabled unless explicitly
   disabled via the helm template variable `sidecarInjectorWebhook.enabled`)
```

Comment:
@ostromart please correct me if I'm wrong, but it seems to me that there are now 3 different config parameters that can be set to disable sidecar injection:

1. autoInjection.enabled=false
2. autoInjection.injector.enabled=false
3. values.sidecarInjectorWebhook.enabled=false

So I would recommend just changing this to say "disabled during installation".

```suggestion
  disabled during installation)
```

Reply:
Yes, that's correct. We do have multiple knobs to do the same thing for the time being due to helm passthrough support - these will be eliminated in the next couple of releases.

Reply:
fixed